src/test/java/seedu/taskify/logic/parser/AddCommandParserTest.java

```diff
@@ -83,57 +74,50 @@ public void parse_compulsoryFieldMissing_failure() {
         String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE);
 
         // missing name prefix
-        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, VALID_NAME_BOB + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB, expectedMessage);
 
         // missing phone prefix
-        assertParseFailure(parser, NAME_DESC_BOB + VALID_DESCRIPTION_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + VALID_DESCRIPTION_BOB
                         + DATE_DESC_BOB, expectedMessage);
 
-        // missing address prefix
-        assertParseFailure(parser,
-                NAME_DESC_BOB + DESCRIPTION_DESC_BOB + VALID_ADDRESS_BOB
-                        + DATE_DESC_BOB, expectedMessage);
 
         // missing date prefix
         assertParseFailure(parser,
-                NAME_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+                NAME_DESC_BOB + DESCRIPTION_DESC_BOB
                         + VALID_DATE_BOB, expectedMessage);
 
         // all prefixes missing
-        assertParseFailure(parser, VALID_NAME_BOB + VALID_DESCRIPTION_BOB + VALID_ADDRESS_BOB
+        assertParseFailure(parser, VALID_NAME_BOB + VALID_DESCRIPTION_BOB
                         + VALID_NAME_BOB, expectedMessage);
     }
 
     @Test
     public void parse_invalidValue_failure() {
         // invalid name
-        assertParseFailure(parser, INVALID_NAME_DESC + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, INVALID_NAME_DESC + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Name.MESSAGE_CONSTRAINTS);
 
         // invalid phone
-        assertParseFailure(parser, NAME_DESC_BOB + INVALID_DESCRIPTION_DESC + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + INVALID_DESCRIPTION_DESC
                         + DATE_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Description.MESSAGE_CONSTRAINTS);
 
-        // invalid address
-        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB + INVALID_ADDRESS_DESC
-                        + DATE_DESC_BOB + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Address.MESSAGE_CONSTRAINTS);
 
         // invalid date
-        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB
                         + INVALID_DATE_DESC + TAG_DESC_HUSBAND + TAG_DESC_FRIEND, Date.MESSAGE_CONSTRAINTS);
 
         // invalid tag
-        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB + ADDRESS_DESC_BOB
+        assertParseFailure(parser, NAME_DESC_BOB + DESCRIPTION_DESC_BOB
                         + DATE_DESC_BOB + INVALID_TAG_DESC + VALID_TAG_FRIEND, Tag.MESSAGE_CONSTRAINTS);
 
         // two invalid values, only first invalid value reported
```

Comment:
Following the comment, the assertParseFailure() in line 115 should have 2 invalid values. Can change either the description or date to an invalid value

Reply:
done